rules.ini

```diff
@@ -80,6 +80,9 @@ RPGMaker[] = \.(?:rgssad|rvproj2|rgss3a|rgss2a)$
 Snowdrop = \.sdfdata$
 Source = (?:^|/)(?:vphysics|bsppack)\.(?:dylib|dll|so)$
 Source2 = (?:^|/)gameinfo\.gi$
+Unigine[] = (?:^|/)(?:|lib)Unigine_x(?:86|64)\.(?:dylib|dll|so)$
+Unigine[] = (?:^|/)core\.ung$
+Unigine[] = (?:^|/)unigine\.cache$
```

Comment:
Cache and core files do not seem to be necessary to detect, as it always finds the binaries.

Reply:
Okay